src/release_linux.go

```diff
@@ -16,7 +20,14 @@ func writeDesktopFiles(mc *metaConfig) {
 }
 
 func configureAutologin(mc *metaConfig) {
-	// set up user autologin
+	if _, err := os.Stat("/usr/share/lightdm/lightdm.conf.d/50-ubuntu.conf"); err == nil {
```

Comment:
You might want to check more generic files, like just the main `lightdm.conf` that is likely to exist on most distros (same with the GDM check, I don't think most distros come with `custom` already existing.

Reply:
Isnt lightdm.conf only on ubuntu 14?

Reply:
I thought it was on all `lightdm` installs, but the `dpkg -S` says otherwise so I guess not. You could check if `/usr/share/lightdm` folder exists, so it would work on other distros as well?

Reply:
Ok, it now checks if directory exists